### ISSUE TITLE
Bug when using `m` as mark

### ISSUE THREAD

**Issue by chrisgrieser** [kometenstaub/obsidian-vim-gutter-marker#8]
I sometimes use  `mm` as a quick mark. However, after doing so, `'mk` to go the the mark and then one line up results in the following `k` being interpreted as a mark. (Vim mode itself works correctly, it seems to be mostly this bug that registers this wrong.)

<img width="517" alt="Pasted image 2022-07-31 17 32 25" src="https://user-images.githubusercontent.com/73286100/182033900-54312bfe-cefc-460f-84ef-ac6b02962f51.png">

**Comment by kometenstaub** [comment]
Yes, that’s a bug with the implementation of the key reading. There may be a way to get them from cm6 vim instead of the current rather hacky way.
